code/components/citizen-server-impl/src/state/ServerGameState_Scripting.cpp

```diff
@@ -1657,6 +1657,27 @@ static void Init()
 
 		return handle;
 	}));
+
+	fx::ScriptEngine::RegisterNativeHandler("GET_PED_STEALTH_MOVEMENT", makeEntityFunction([](fx::ScriptContext& context, const fx::sync::SyncEntityPtr& entity)
+	{
+		auto movementGroup = entity->syncTree->GetPedMovementGroup();
+
+		return movementGroup ? movementGroup->isStealthy : false;
+	}));
+
+	fx::ScriptEngine::RegisterNativeHandler("IS_PED_FREE_AIMING", makeEntityFunction([](fx::ScriptContext& context, const fx::sync::SyncEntityPtr& entity)
+	{
+		auto movementGroup = entity->syncTree->GetPedMovementGroup();
+
+		return movementGroup ? movementGroup->isStrafing : false;
```

Comment:
Are you sure that `isStrafing` here makes sense for `IS_PED_FREE_AIMING` native?

Reply:
That’s the name that I found using Xbox dump, but yes if needed I could change it

Reply:
Hmm this is weird, are you sure this variable is not used for anything else instead? Like strafing or so?

Reply:
Changed native name to `IS_PED_STRAFING` after some tests.